Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11.4-alpine
+LABEL maintainer="volodymyr_dolgyi@ukr.net"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]

```

Comment:
The task description specifies that the Docker image should be published to Docker Hub. Please ensure that you include the necessary steps or instructions to publish the image.